test/resource-limits.test.ts

```diff
@@ -71,7 +74,7 @@ describe.each(['worker_threads', 'child_process'] as const)('%s', (runtime) => {
     }
 
     // Test setup should not reach max memory on first round
-    expect(rounds).toBeGreaterThan(1)
+    expect(rounds).toBeGreaterThan(0)
```

Comment:
> Test setup should not reach max memory on first round
The comment does not match with expectation. Initial value is `0` so any value greater than 0 will make sure it's not crashed in first round.